portal-web/test/functional/com/liferay/portalweb/macros/MultiFactorAuthentication.macro

```diff
@@ -304,6 +305,14 @@ definition {
 
 			AssertElementNotPresent(locator1 = "MultiFactorAuthentication#EMAIL_OTP_VERIFY_FIELD");
 
+			if (IsElementPresent(locator1 = "Button#I_AGREE")) {
```

Comment:
The new added user is required password reset, prefer to use JSONUser.setFirstPassword after adding new user. Then no need to add Line308-315.